File: C Programming/Assignments/Assignment 6/func with no return type and with para/Assignment 1/Q4/Q4.c
//Write a program to check whether a given character is a vowel or consonant.
//(vowel - a,e,i,o,u,A,E,I,O,U others are consonant)

#include<stdio.h>

void checkVowelConsonant(char);
void main(){
	
	//variable initialization
	char ch='A';
	
	checkVowelConsonant(ch);
	
}

void checkVowelConsonant(char ch){

	//applying condition
	if(ch=='a'||ch=='A'||ch=='e'||ch=='E'||ch=='i'||ch=='I'||ch=='o'||ch=='O'||ch=='u'||ch=='U')
		printf("%c is a vowel",ch);
	else
		printf("%c is a consonant",ch);
	
}

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 1/Q4/Q4.java
//Write a program to check whether a given character is a vowel or consonant.
//(vowel - a,e,i,o,u,A,E,I,O,U others are consonant)

public class Q4 {
    public static void main(String[] args) {
        // variable initialization
        char ch = 'A';

        // applying condition
        if (ch == 'a' || ch == 'A' || ch == 'e' || ch == 'E' || ch == 'i' || ch == 'I' || ch == 'o' || ch == 'O' || ch == 'u' || ch == 'U')
            System.out.println(ch + " is a vowel");
        else
            System.out.println(ch + " is a consonant");
    }
}


